### PR TITLE
fix: secondary sentiments on some components

### PR DESCRIPTION
**Pull request by fabienhebert**
## Type

- Enhancement

### Summarise concisely:

We added a `secondary` color/sentiment to the THEME however if wasn't accepted by some components

### CODE REVIEW

File: packages/ui/src/components/Text/index.tsx [hunk 1]
```diff
@@ -159,7 +166,7 @@ export const Text = ({
         as={as}
         placement={placement}
         prominence={prominence}
-        color={color}
+        color={computedSentiment}
```
Comment (QuiiBz) [inline packages/ui/src/components/Text/index.tsx:169]: Should we rename the prop too?

File: packages/ui/src/components/Badge/index.tsx [hunk 2]
```diff
@@ -158,28 +158,53 @@ export const Badge = ({
...
+        <StyledText
+          variant="bodyStrong"
+          as="p"
+          fontSize={TEXT_SIZES[size]}
+          prominence={disabled ? 'weak' : 'default'}
+        >
+          {children}
+        </StyledText>
+      </StyledBox>
+      <StyledBox
```
Comment (QuiiBz) [inline packages/ui/src/components/Badge/index.tsx:185]: I'm not sure to understand why it's being duplicated?

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/Text/index.tsx (hunk 1 above)
Comment (fabienhebert) [reply packages/ui/src/components/Text/index.tsx:169]: It's internal to the component but I can check indeed

File: packages/ui/src/components/Badge/index.tsx (hunk 2 above)
Comment (fabienhebert) [reply packages/ui/src/components/Badge/index.tsx]: i forgot to remove this test 🙈

**Review by QuiiBz — APPROVED** [review]

**Review by philibea — APPROVED** [review]

**Pull request merged**